assets/js/script.js

```diff
@@ -0,0 +1,151 @@
+// Assignment Code
+var generateBtn = document.querySelector("#generate");
+// array for uppercase,lowercase,special characters and numbers.
+const numArray = ["0", "1", "2", "3", "4", "5", "6", "7", "8", "9"];
+const specialArray = [
+  "!",
+  "£",
+  "$",
+  "%",
+  "^",
+  "&",
+  "*",
+  "(",
+  ")",
+  "_",
+  "-",
+  "+",
+  "=",
+  "?",
+  "@",
+  "#",
+  "~",
+  "|",
+  "/",
+  ".",
+  ",",
+  "<",
+  ">",
+  "'",
+  ":",
+  " ",
+  ";",
+  "`",
+  "¬",
+];
+const lowerCaseArray = [
+  "a",
+  "b",
+  "c",
+  "d",
+  "e",
+  "f",
+  "g",
+  "h",
+  "i",
+  "j",
+  "k",
+  "l",
+  "m",
+  "n",
+  "o",
+  "p",
+  "q",
+  "r",
+  "s",
+  "t",
+  "u",
+  "v",
+  "w",
+  "x",
+  "y",
+  "z",
+];
+
+// used a formula for uppercase array
+const upperCase = lowerCaseArray.join("").toUpperCase();
+const upperCaseArray = upperCase.split("");
+
+// This gets the length and what the user decides to include in their password (lowercase/uppercase/special/num)
+const getCriteria = function () {
+  // prompt for user to decide the length of their password
+  const x = prompt("Choose Password length");
+  // convert the input from a string into an int
```

Comment:
As mentioned in the feedback, I think it might be worth adding in the character limits here so the user doesn't have to hit the alert to find out the character limits.